webapp/app/services/roleService.js

```diff
@@ -1,17 +1,43 @@
 define('services/roleService', [], function () {
 
     function roleReadAll() {
-    // TODO: refactor common headers and base url
-    return $.ajax({
-      type: 'POST',
-      contentType: 'application/json',
-      url:'http://localhost:10160/Service.svc/jsonp/post',
-      data: '{"requests":[{"__type":"RoleReadAllRequest:#P202.Training.WCF.RequestsAndResponses"}]}'
-    });
-  }
+        console.log('Entre a llamar WebService')
+        return $.ajax({
+            type: 'POST',
+            contentType: 'application/json',
+            url: 'http://localhost:10160/Service.svc/jsonp/post',
+            data: '{"requests":[{"__type":"RoleReadAllRequest:#P202.Training.WCF.RequestsAndResponses"}]}'
+        });
+    }
 
-  return {
-      roleReadAll: roleReadAll
-  };
+    function roleAdd(value) {
+        var roleIns = {
+            "Name":value
+        };              
+        var roleInsJs = JSON.stringify(roleIns);
```

Comment:
you can replace the previous 4 lines with sth like `var role = '{ "Name": ' + roleName + '}';`

Reply:
the lines was replace with only one line of code